packages/coding-standards/examples/ValidClass.php

```diff
@@ -10,10 +10,13 @@
 final class ValidClass
 {
     /**
-     * @var Type
+     * @var \ShopsysNamespace\Type
      */
     private $parameterCamelCase;
 
+    /**
+     * @param ShopsysNamespace\Type $parameterCamelCase
```

Comment:
Yes, this should be outdated (see yellow "oudated" above). 

I guess you're jumping commit by commit so you've noticed already fixed spots.

I'll try to squash these to one next time, so that can't happen again.